MyApplication/app/src/main/java/com/joelimyx/myapplication/MainActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.joelimyx.myapplication;
+
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.app.NotificationCompat;
+import android.view.View;
+import android.widget.Button;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener{
+    private Button mPlay, mStop, mPause;
+    private boolean isPlay = true;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        mPlay = (Button) findViewById(R.id.play);
+        mStop = (Button) findViewById(R.id.stop);
+        mPlay.setOnClickListener(this);
+        mStop.setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View view) {
+        Intent intent = new Intent(MainActivity.this,MediaService.class);
+        switch (view.getId()){
+            case R.id.play:
+                if (isPlay) {
+                    mPlay.setText("Pause");
+                    isPlay = false;
+                }else{
+                    mPlay.setText("Play");
+                    isPlay = true;
+                    intent.putExtra("Pause",true);
```

Comment:
You might want to use `setAction()` rather than `putExtra()`, then you won't have to bother with a value you don't need.